### PR TITLE
Update maven wrapper

### PR DESCRIPTION
**Pull request by elharo**
## Description
Ran mvn wrapper:wrapper to update to the latest maven wrapper.

## Motivation and Context
Move maven wrapper to the recommended 3.8.8 version of Maven

## Impact
none

## Test Plan
CI

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by elharo** [comment]
2024-03-29T21:48:09.7024875Z Apache Maven 3.8.8 (4c87b05d9aedce574290d1acc98575ed5eb6cd39)
2024-03-29T21:48:09.7026639Z Maven home: /home/runner/.m2/wrapper/dists/apache-maven-3.8.8-bin/6604843/apache-maven-3.8.8
2024-03-29T21:48:09.7028081Z Java version: 1.8.0_402, vendor: Azul Systems, Inc., runtime: /opt/hostedtoolcache/jdk/8.0.402/x64/jre
2024-03-29T21:48:09.7029203Z Default locale: en, platform encoding: UTF-8
2024-03-29T21:48:09.7030174Z OS name: "linux", version: "6.5.0-1016-azure", arch: "amd64", family: "unix"
2024-03-29T21:48:10.3345361Z [ERROR] Error executing Maven.
2024-03-29T21:48:10.3346716Z [ERROR] Invalid threads value: 'C1'. Supported are int values.

**Comment by elharo** [comment]
2024-03-29T22:21:55.0412787Z   MAVEN_TEST: -B -Dair.check.skip-all -Dmaven.javadoc.skip=true -DLogTestDurationListener.enabled=true --no-transfer-progress --fail-at-end
2024-03-29T22:21:55.0413801Z   RETRY: .github/bin/retry
2024-03-29T22:21:55.0414133Z ##[endgroup]
2024-03-29T22:21:57.7367308Z ##[error]Unexpected HTTP status code '500' when retrieving versions from 'https://static.azul.com/zulu/bin/'. <!DOCTYPE html>



 <html class="no-js" lang="en-US"> 
<head>
<title>Worker threw exception | static.azul.com | Cloudflare</title>
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" />

<style>body{margin:0;padding:0}</style>



<script>
  if (!navigator.cookieEnabled) {
    window.addEventListener('DOMContentLoaded', function () {
      var cookieEl = document.getElementById('cookie-alert');
      cookieEl.style.display = 'block';
    })
  }
</script>



</head>
<body>
  <div id="cf-wrapper">
    <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
    <div id="cf-error-details" class="cf-error-details-wrapper">
      <div class="cf-wrapper cf-header cf-error-overview">
        <h1>
          <span class="cf-error-type" data-translate="error">Error</span>
          <span class="cf-error-code">1101</span>
          <small class="heading-ray-id">Ray ID: 86c34173aeff594a &bull; 2024-03-29 22:21:57 UTC</small>
        </h1>
        <h2 class="cf-subheadline" data-translate="error_desc">Worker threw exception</h2>
      </div>

      <section></section>

      <div class="cf-section cf-wrapper">
        <div class="cf-columns two">
          <div class="cf-column">
            <h2 data-translate="what_happened">What happened?</h2>
            <p>You've requested a page on a website (static.azul.com) that is on the <a href="https://www.cloudflare.com/5xx-error-landing/" target="_blank">Cloudflare</a> network. An unknown error occurred while rendering the page.</p>
          </div>

          
          <div class="cf-column">
            <h2 data-translate="what_can_i_do">What can I do?</h2>
            <p><strong>If you are the owner of this website:</strong><br />you should <a href="https://www.cloudflare.com/login?utm_source=error_100x" target="_blank">login to Cloudflare</a> and check the error logs for static.azul.com.</p>
          </div>
          
        </div>
      </div>

      <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
  <p class="text-13">
    <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">86c34173aeff594a</strong></span>
    <span class="cf-footer-separator sm:hidden">&bull;</span>
    <span id="cf-footer-item-ip" class="cf-footer-item hidden sm:block sm:mb-1">
      Your IP:
      <button type="button" id="cf-footer-ip-reveal" class="cf-footer-ip-reveal-btn">Click to reveal</button>
      <span class="hidden" id="cf-footer-ip">40.76.119.166</span>
      <span class="cf-footer-separator sm:hidden">&bull;</span>
    </span>
    <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing" id="brand_link" target="_blank">Cloudflare</a></span>
    
  </p>
  <script>(function(){function d(){var b=a.getElementById("cf-footer-item-ip"),c=a.getElementById("cf-footer-ip-reveal");b&&"classList"in b&&(b.classList.remove("hidden"),c.addEventListener("click",function(){c.classList.add("hidden");a.getElementById("cf-footer-ip").c
2024-03-29T22:21:57.7474985Z Post job cleanup.

**Comment by elharo** [comment]
Looks like the failure was flaky

File: .circleci/continue_config.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ executors:
     environment:
       MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
       MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
-      MAVEN_FAST_INSTALL: "-B -V --quiet -T C1 -DskipTests -Dair.check.skip-all -Dmaven.javadoc.skip=true"
```
Comment (tdcmeehan) [inline .circleci/continue_config.yml:69]: Just to make sure I understand, the format of this argument changed in between Maven versions?

**Review by tdcmeehan — APPROVED** [review]

File: .circleci/continue_config.yml (hunk 1 above)
Comment (elharo) [reply .circleci/continue_config.yml:69]: Yes, exactly. I think 1C (one core) always worked but C1 (core one) used to work and no longer does.

**Pull request merged**